### PR TITLE
[Chore] Next level

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Update PromiseTracker to the latest Nextjs and introduce Sentry integration.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Screenshots

Screenshot from 2021-06-22 08-34-26

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/pages/_document.js [hunk 1]
```diff
@@ -13,6 +11,10 @@ export default class MyDocument extends Document {
         <Head>
           {/* PWA primary color */}
           <meta name="theme-color" content={theme.palette.primary.main} />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Amiri:ital,wght@0,400;0,700;1,400&family=Open+Sans:wght@300;400;600;700&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap"
+            rel="stylesheet"
+          />
```
Comment (KhadijaMahanga) [inline src/pages/_document.js:17]:
```suggestion
          <link
            href="https://fonts.googleapis.com/css2?family=Amiri:ital,wght@0,400;0,700;1,400&family=Open+Sans:wght@300;400;600;700&family=Source+Sans+Pro:wght@200;300;400;600;700&display=swap"
            rel="stylesheet"
          />
          <link
            rel="preconnect"
            href="https://fonts.googleapis.com"
            crossOrigin="anonymous"
          />
```

**Review by KhadijaMahanga — APPROVED** [review]
Clean 🚀

**Review by KobbyMmo — COMMENTED** [review]
Looks good to me.
Not related to this PR though, but I think promises are not being pulled from Check.

**Comment by kilemensi** [comment]
> Looks good to me.
> Not related to this PR though, but I think promises are not being pulled from Check.

Are they not loading @KobbyMmo or there are no verified Promises in Check?... And where is my ✅ ?

**Review by KobbyMmo — APPROVED** [review]
looks good to me

File: src/pages/_document.js (hunk 1 above)
Comment (kilemensi) [reply src/pages/_document.js:17]: Someone _**had**_ to find something..

**Pull request merged**